tlp-functions.in

```diff
@@ -1853,6 +1853,24 @@ set_usb_suspend () { # activate usb autosuspend for all devices except input and
                         fi # phone
 
                         if [ -z "$exc" ]; then
+                            # check for printers
+                            USB_BLACKLIST_PRINTER=${USB_BLACKLIST_PRINTER:-1} # default is exclude
+
+                            if [ "$USB_BLACKLIST_PRINTER" = "1" ]; then
+                                if [ "$dclass" != "00" ]; then
```

Comment:
I checked the [USB Printer Class Spec](http://www.usb.org/developers/docs/devclass_docs/usbprint11a021811.pdf). On p.8 it says that bDeviceClass == 00h, so your check will be negative. I suggest you remove it.